app/screens/ScheduleScreen/ScheduleCard.tsx

```diff
@@ -162,11 +188,19 @@ const ScheduleCard: FC<ScheduleCardProps> = (props) => {
             />
           ),
         }
-      : baseSpeakingEventProps({ heading, subheading, eventTitle, sources })
+      : baseSpeakingEventProps({ heading, subheading, eventTitle, sources, isPast })
 
   return (
     <Card
-      preset={variant === "recurring" || variant === "party" ? "reversed" : "default"}
+      preset={
+        isPast
+          ? variant === "recurring" || variant === "party"
```

Comment:
This is totally fine, and correct, but if you wanted to make it just a bit more readable, you could do something like this:

```
const isReversed = variant === "recurring" || variant === "party"
const cardPreset = isReversed ? ( isPast ? "pastReversed" : "reversed") : ( isPast ? "pastDefault" : "default")
```

Definitely optional!

Reply:
Smart! Will update it!